internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -325,6 +357,28 @@ func resourceAppConfigurationCreate(d *pluginsdk.ResourceData, meta interface{})
 	}
 	meta.(*clients.Client).AppConfiguration.AddToCache(resourceId, *resp.Model.Properties.Endpoint)
 
+	expandedReplicas, err := expandAppConfigurationReplicas(d.Get("replica").(*schema.Set).List(), name, location)
+	if err != nil {
+		return fmt.Errorf("expanding `replica`: %+v", err)
+	}
+
+	replicaClient := meta.(*clients.Client).AppConfiguration.ReplicasClient
+	for _, replica := range expandedReplicas {
+		replicaId := replicas.NewReplicaID(subscriptionId, resourceGroup, name, *replica.Name)
+
+		existingReplica, err := replicaClient.Get(ctx, replicaId)
+		if err != nil {
+			if !response.WasNotFound(existingReplica.HttpResponse) {
+				return fmt.Errorf("retrieving %s: %+v", replicaId, err)
+			}
+		}
```

Comment:
We probably don't need to check if any replica's already exist here since this is in the create and we check this when we get the resource?
```suggestion
```

Reply:
fixed, thanks.